requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests>=2.27.1
```

Comment:
1. alphabetic order
2. no testing dependencies in requirements.txt. Instead, they should be in requirements_qa.txt
3. better to freeze exact versions (not mandatory)